packages/data-context/src/actions/ProjectActions.ts

```diff
@@ -511,4 +513,20 @@ export class ProjectActions {
       }
     })
   }
+
+  async pingBaseUrl () {
+    const baseUrl = (await this.ctx.project.getConfig())?.baseUrl
+
+    // Should never happen
+    if (!baseUrl) {
+      throw new Error('Cannot ping url without a valid baseUrl')
```

Comment:
Out of curiosity, what ends up happening in the app if this error is thrown?

Reply:
Tested and the error was getting swallowed, a noop makes sense.

Reply:
Done in fa94e7e98e3ab17e1a46eb7687b88ee27321a9d2